distributed/deploy/ssh.py

```diff
@@ -45,6 +45,9 @@ async def close(self):
     def __repr__(self):
         return f"<SSH {type(self).__name__}: status={self.status}>"
 
+    def _repr_html_(self, heading=None):
+        return super()._repr_html_(heading=heading)
```

Comment:
This doesn't really seem necessary as `heading` is not being set and nothing different is being done in this method.

I would be tempted to just remove it and allow the `super` to handle it.

Reply:
Added :heavy_check_mark: